### PR TITLE
Add Auditbeat dashboards (overview, sockets, execs)

### PR DESCRIPTION
**Pull request by andrewkroh**
Add dashboards to view events generated by the Linux audit framework. This PR adds three dashboards:

- Overview - A general dashboard showing a summary of all events. Executions
- Executions - A dashboard showing process executions (`execve` and `execveat`) syscalls. Executions
- Sockets - A dashboard showing information related sockets and remote connectivity (e.g. `bind`, `connect`, `accept`, `recvfrom`). Sockets

### CODE REVIEW

**Comment by andrewkroh** [comment]
Without adding the proper rules the execution and socket dashboards will show no data. The overview dashboard will likely contain some events because even with no audit rules installed there are some events generated.

**Comment by andrewkroh** [comment]
Should this be back-ported to 6.0 or is it too late?

File: auditbeat/auditbeat.reference.yml [hunk 1]
```diff
@@ -40,10 +40,30 @@ auditbeat.modules:
...
-    # Create file watches (-w) or syscall audits (-a or -A). For example:
+    ## Define audit rules here.
+    ## Create file watches (-w) or syscall audits (-a or -A). Uncomment these
+    ## examples or add your own rules.
+
+    ## If you are on a 64 bit platform, everything should be running
+    ## in 64 bit mode. This rule will detect any use of the 32 bit syscalls
+    ## because this might be a sign of someone exploiting a hole in the 32
+    ## bit API.
+    #-a always,exit -F arch=b32 -S all -F key=32bit-abi
```
Comment (tsg) [inline auditbeat/auditbeat.reference.yml]: Awesome that we've got these.

**Comment by andrewkroh** [comment]
Will add the 5.x conversions in a separate PR.

**Pull request merged**